lib/scrabble.rb

```diff
@@ -1,4 +1,92 @@
 module Scrabble
+
+LETTER_VALUES = [
+  [1, "A", "E", "I", "O", "U", "L", "N", "R", "S", "T"],
```

Comment:
Based on the way you have structured this - it might make sense to use a hash since it seems like the first element in the array index is your "key"